evd_ros_core/RobotMove goal
---
bool status